src/components/ContentSidebar/Sidebar.js

```diff
@@ -9,7 +9,6 @@ import DetailsSidebar from './DetailsSidebar';
 import SkillsSidebar from './SkillsSidebar';
 import ActivitySidebar from './ActivitySidebar';
 import MetadataSidebar from './MetadataSidebar';
```

Comment:
This file can also be reverted if desired, but if reverting, then the render() function of ContentSidebar.js should just send in the props needed by this file